### PR TITLE
Creation of SIG/Google Group for competitions and student developers

### PR DESCRIPTION
**Pull request by boronhub**
A SIG for student or beginner developers or competition participants to interact with each other and foster a spirit of open source development.

NOTE; I have not specified a link to a mailing list or the project leads, as I wish to discuss those with the GCI mentors and community maintainers.

Edit: Check out [this ](https://groups.google.com/a/tensorflow.org/forum/#!topic/discuss/jvQzUiqw_6g)post for more context.

### CODE REVIEW

**Comment by perfinion** [comment]
I don't think SIG-Community makes sense as a name. All SIGs are community groups, this isnt any more community than the others, if anything this should be like SIG-Students or something like that?

But more generally, I think this is more suited for TensorFlow User Groups and less for a SIG. Eg: https://www.tensorflow.org/community/groups

File: sigs/community/CHARTER.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Proposed name: SIG Community
+
+## Objective
+
+A platform for students and new developers looking to get into TensorFlow, as well
```
Comment (terrytangyuan) [inline sigs/community/CHARTER.md]: We used Gitter for communications during last year's GSoC and I found it very sufficient already. Why would a platform and a new SIG be necessary for objective like this?

Comment (boronhub) [reply sigs/community/CHARTER.md]: Maybe GSoC had a platform, but GCI didn't.

**Comment by boronhub** [comment]
Alright, I see how a group would be better suited. How do we go about creating that?

**Comment by WilliamHYZhang** [comment]
This should probably wait until next year's GCI, let's leave it to the admins.

**Comment by bhack** [comment]
@theadactyl I think that this is probbly out of scope for a SIG. Please comment or route it if you can.

**Comment by dotrachit** [comment]
oh

**Comment by theadactyl** [comment]
Hi all! Apologies for the very late response. This is Out of Scope for the SIG program, which focuses on particular areas of the code. (We decided many moons ago to not have "meta SIGs".)

I think that this could be a good topic or subcategory to have on the TensorFlow Forum - discuss.tensorflow.com. I encourage you to start a thread there in the Meta category if this is something you'd still like to do. Since this isn't a code related question, I'm going to close this PR.

**Pull request closed without merging**